app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheet.kt

```diff
@@ -133,7 +133,7 @@ class ScheduleAppointmentSheet : BottomSheetActivity() {
       is TimeToAppointment.Months -> R.plurals.scheduleappointment_appointmentin_months
     }
 
-    return resources.getQuantityString(quantityStringResourceId, timeToAppointment.value, timeToAppointment.value)
+    return resources.getQuantityString(quantityStringResourceId, timeToAppointment.value, "${timeToAppointment.value}")
```

Comment:
Unnecessary template?

Reply:
As discussed, I am not making this change.